r/DESCRIPTION

```diff
@@ -50,6 +49,8 @@ Suggests:
     rmarkdown,
     testthat,
     tibble
+Remotes:
+    r-lib/cpp11#97
```

Comment:
When we're ready to merge this, we should vendor cpp11. I made https://issues.apache.org/jira/browse/ARROW-9786 to go back to the released cpp11 before we do the 2.0.0 release (October-ish).